test/homepage-noweb-test.js

```diff
@@ -41,24 +40,31 @@ suite.addBatch({
         },
         "and we visit the root URL": {
             topic: function() {
-                var browser;
-                browser = new Browser();
+                var cb=this.callback,
+                    browser = new Browser();
 
-                browser.visit("http://localhost:4815/", this.callback);
+                browser.visit("http://localhost:4815/", function(){
+                        cb(!browser.success, browser);
```

Comment:
OK, I see what you're doing now. However, passing a boolean to an argument named `err`, and that originally contained a _real_ error object, seem inelegant. Is there a better way to do this?